content/spin/rust-components.md

```diff
@@ -106,12 +106,12 @@ id = "hello"
 source = "target/wasm32-wasi/release/spinhelloworld.wasm"
 allowed_http_hosts = [ "some-random-api.ml" ]
 [component.trigger]
-route = "/hello"
+route = "/wilcard"
```

Comment:
I think the code here more closely follows the `/outbound` example (https://github.com/fermyon/spin/blob/013059cb7eda1189190fdec9fced86c9fc88867f/examples/http-rust-outbound-http/spin.toml#L11) - suggest using that rather than `wildcard`

Reply:
Agreed, thanks Ivan.